alf/environments/suite_unittest_test.py

```diff
@@ -60,10 +67,13 @@ def test_value_unittest_env(self):
                                            dtype=tf.int64)
                 time_step = env.step(action)
 
-    def test_policy_unittest_env(self):
+    @parameterized.parameters(False, True)
```

Comment:
It is simpler and easier to read to use Action.Discrete and Action.Continuous as the parameter.